homeassistant/components/sensor/dht.py

```diff
@@ -109,8 +120,7 @@ def update(self):
             if (temperature >= -20) and (temperature < 80):
                 self._state = temperature
                 if self.temp_unit == TEMP_FAHRENHEIT:
-                    self._state = round(celsius_to_fahrenheit(temperature),
-                                        1)
+                    self._state = round(celsius_to_fahrenheit(temperature), 1)
```

Comment:
Maybe but I can't really test such a change at the moment. I opened a [story](https://www.pivotaltracker.com/story/show/129707403) for a possible move to use the unit system information in a larger scale.
I would prefer to stick to the configuration change and a little cleanup for now.